### PR TITLE
Removing GOOGLE_TAG_MANAGER_PREVIEW

### PR DESCRIPTION
**Pull request by pricem14pc**
Following removal of the `gtm_preview` request parameter in the GTM snippet it has been confirmed we no longer need to support this value in Runner and references to it can be removed.

Ensure the environment variable no longer exists and any previous references to it through the HTML templates, the template helpers, and helm deployments have been removed.

The application should deploy, and GTM integration should function as expected, without this value.

### CODE REVIEW

File: ci/deploy_app.yaml [hunk 1]
```diff
@@ -26,6 +25,12 @@ params:
   MIN_REPLICAS:
   MAX_REPLICAS:
   TARGET_CPU_UTILIZATION_PERCENTAGE:
+  CDN_URL:
+  WEB_SERVER_TYPE:
+  WEB_SERVER_WORKERS:
+  WEB_SERVER_THREADS:
+  WEB_SERVER_UWSGI_ASYNC_CORES:
+  DATASTORE_USE_GRPC:
```
Comment (pricem14pc) [inline ci/deploy_app.yaml:33]: Added these as part of this PR as they were previously missing. While it doesn't impact automated pipelines it does mean they can now be overridden by a local developer fly exec to a GCP env which is a useful approach.

File: templates/layouts/_base.html [hunk 2]
```diff
@@ -94,10 +94,10 @@
 
 
 {% block bodyStart %}
-  {% if google_tag_manager_id and google_tag_manager_auth and google_tag_manager_preview  %}
+  {% if google_tag_manager_id and google_tag_manager_auth  %}
     <!-- Google Tag Manager (noscript) -->
     <noscript>
-      <iframe src="https://www.googletagmanager.com/ns.html?id={{ google_tag_manager_id }}&gtm_auth={{ google_tag_manager_auth }}&gtm_preview={{ google_tag_manager_preview }}&gtm_cookies_win=x"
+      <iframe src="https://www.googletagmanager.com/ns.html?id={{ google_tag_manager_id }}&gtm_auth={{ google_tag_manager_auth }}&&gtm_cookies_win=x"
```
Comment (pricem14pc) [inline templates/layouts/_base.html:100]: This was not (but could have been) removed in the previous PR as it was not part the testing.

Comment (ajmaddaford) [inline templates/layouts/_base.html:100]:
```suggestion
      <iframe src="https://www.googletagmanager.com/ns.html?id={{ google_tag_manager_id }}&gtm_auth={{ google_tag_manager_auth }}&gtm_cookies_win=x"
```

**Review by MebinAbraham — APPROVED** [review]
Apart from Andrew's comment, seems to be as expected 👍

**Review by ajmaddaford — APPROVED** [review]

**Pull request merged**